app/views/forever_style_guide/demo/hero_simple.html.erb

```diff
@@ -0,0 +1,47 @@
+<header id="header-is_fixed">
+  <%= render partial: "layouts/forever_style_guide/navigation" %>
+</header>
+
+<div class="hero_simple hero_simple-sample">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-xs-12 text-center hero_simple-block hero_simple-block-secondary hero_simple-block_description">
+        <h1 class="page-title">Now presenting .hero_simple</h1>
+        <p>Available in all the standard brand colors for (hopefully) easy implementation.</p>
+        <%= link_to 'Primary Action', "#", class: 'btn btn-lg btn-primary hero_simple-block-cta', title: "Primary Action" %>
+        <%= link_to 'Learn More', "#", class: 'btn btn-lg btn-artisan btn-inverse hero_simple-block-cta ', title: "Learn More" %>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="clearfix l-padded"></div>
+
+<div class="hero_simple hero_simple-sm color_block-services">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="text-center l-padded hero_simple-sm-block">
```

Comment:
there's no columns w/ in the row for the hero_simple-sm examples. Is that an intentional difference between the "standard" hero simple markup?